cmd/instances.go

```diff
@@ -272,6 +274,52 @@ settings for a specified domain.
 	},
 }
 
+var updateInstancePassphraseCmd = &cobra.Command{
+	Use:     "set-passphrase <domain> <current-passphrase> <new-passphrase>",
```

Comment:
Hmmm, I wouldn't have asked for the current passphrase on this command, just the new one. If you have access to the CLI, you have special access rights and you are expected to know what you are doing.